selfdrive/car/mazda/carcontroller.py

```diff
@@ -0,0 +1,91 @@
+from selfdrive.car.mazda import mazdacan
+from selfdrive.car.mazda.values import CAR, DBC
+from selfdrive.can.packer import CANPacker
+from selfdrive.car import apply_std_steer_torque_limits
+
+
+class CarControllerParams():
+  def __init__(self, car_fingerprint):
+    self.STEER_MAX = 600                 # max_steer 2048
+    self.STEER_STEP = 1                  # how often we update the steer cmd
+    self.STEER_DELTA_UP = 10             # torque increase per refresh
+    self.STEER_DELTA_DOWN = 20           # torque decrease per refresh
+    if car_fingerprint == CAR.CX5:
```

Comment:
not needed if

Reply:
I left it there just to make it clear for the next guy who port a new mazda to tweak it, although all current gen Mazda seems to be the same. The new Mazda3 could be different.